packages/router/src/active-route-loader.tsx

```diff
@@ -12,15 +12,19 @@ interface Props {
   children?: React.ReactNode
 }
 
-let isPrerendered = false
+const isPrerendered = false
 
-if (typeof window !== 'undefined') {
-  const redwoodAppElement = document.getElementById('redwood-app')
+// @MARK @TODO
+// SSR and streaming changes how we mount the React app (we render the whole page, including head and body)
+// This logic is no longer valid and needs to be rethought
 
-  if (redwoodAppElement && redwoodAppElement.children.length > 0) {
-    isPrerendered = true
-  }
-}
+// if (typeof window !== 'undefined') {
```

Comment:
`vite/index.ts` `define` a variable we can use to check if we're running the streaming experiment